library/objective-c/EnvoyEngineImpl.h

```diff
@@ -0,0 +1,33 @@
+#import "EnvoyEngine.h"
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// Wrapper layer for calling into Envoy's C/++ API.
+@interface EnvoyEngineImpl : NSObject <EnvoyEngine>
+
+/**
+ Run the Envoy engine with the provided config and log level.
+
+ @param config The configuration file with which to start Envoy.
+ @return A status indicating if the action was successful.
+ */
++ (EnvoyStatus)runWithConfig:(NSString *)config;
```

Comment:
Do these need to be in the header file if this is already defined to conform to `EnvoyEngine` above?

Reply:
nope, guess not :)